src/internal/components/LayoutEditor.tsx

```diff
@@ -63,7 +65,26 @@ export const LayoutEditor = (props: LayoutEditorProps) => {
    */
   useEffect(() => {
     devLogger.logData("THEME_DATA", themeData);
```

Comment:
Log the data actually being used, aka have a logger for the localStorage data and move this to the if themeData conditional.